src/components/FullTextView/FullTextView.spec.tsx

```diff
@@ -32,6 +33,22 @@ describe('FullTextView', () => {
     expect(component.find('[data-hook="popover-content"]').at(0).text()).toBe('Delete this super awesome thing?');
   });
 
+  it('should dynamically load and display full content on hover and hide it on leave in tooltip using `shouldLoadAsync` prop', async () => {
+    const content = (
+      <div>
+        <span>Delete this super awesome thing</span>
+        <i>?</i>
+      </div>
+    );
+    const component = mount(<FullTextView shouldLoadAsync maxWidth={0}>{content}</FullTextView>);
+
+    expect(component.find('[data-hook="popover-content"]').length).toBe(0);
+    component.setState({isEllipsisActive: true});
+    await sleep(0);
```

Comment:
eventually

Reply:
`sleep` should be more performant, since we know when to run next check - on the next tick.
but I'm ok with eventually :)

Reply:
I prefer checking "like a user", meaning I wait for it to happen and I don't know when.
for unit-testing the `Loadable` component that would probably make sense, but in more complex cases I'm not sure this would be the case